eos/b-decays/b-to-dstar-l-nu_TEST.cc

```diff
@@ -0,0 +1,156 @@
+
+                const double eps = 1e-5;
+
+                // test for different s-bins - the default lepton is muon
+                TEST_CHECK_RELATIVE_ERROR(d.normalized_integrated_branching_ratio(4.0, 8.0), 15.8018, eps);
+                TEST_CHECK_RELATIVE_ERROR(d.normalized_integrated_branching_ratio(8.0, 10.68), 6.15469, eps);
+                // the full phase-space region for muon
+                TEST_CHECK_RELATIVE_ERROR(d.normalized_integrated_branching_ratio(0.011164, 10.68), 40.4532, eps);
+            }
+
+            // tests for normalized NP Br (|V_cb|=1)
+            {
+                Parameters p2 = Parameters::Defaults();
+                /*
+                 * for the TEST case below the B->D^* SSE parameters are randomly chosen. However, the correlations together with the EOM conditions among the FFs are respected in this choice.
+                 */
+                p2["B->D^*::alpha^A0_0@BSZ2015" ] = 1.0;
+                p2["B->D^*::alpha^A0_1@BSZ2015" ] = 0.0;
+                p2["B->D^*::alpha^A0_2@BSZ2015" ] = 0.0;
+                p2["B->D^*::alpha^A1_0@BSZ2015" ] = 0.0;
+                p2["B->D^*::alpha^A1_1@BSZ2015" ] = 0.0;
+                p2["B->D^*::alpha^A1_2@BSZ2015" ] = 0.0;
+                p2["B->D^*::alpha^A12_0@BSZ2015"] = 0.28064568285509967;
+                p2["B->D^*::alpha^A12_1@BSZ2015"] = 0.0;
+                p2["B->D^*::alpha^A12_2@BSZ2015"] = 0.0;
+                p2["B->D^*::alpha^V_0@BSZ2015"  ] = 0.0;
+                p2["B->D^*::alpha^V_1@BSZ2015"  ] = 0.0;
+                p2["B->D^*::alpha^V_2@BSZ2015"  ] = 0.0;
+                p2["B->D^*::alpha^T1_0@BSZ2015" ] = 0.0;
+                p2["B->D^*::alpha^T1_1@BSZ2015" ] = 0.0;
+                p2["B->D^*::alpha^T1_2@BSZ2015" ] = 0.0;
+                p2["B->D^*::alpha^T2_0@BSZ2015" ] = 0.0;
+                p2["B->D^*::alpha^T2_1@BSZ2015" ] = 0.0;
+                p2["B->D^*::alpha^T2_2@BSZ2015" ] = 0.0;
+                p2["B->D^*::alpha^T23_0@BSZ2015"] = 0.0;
+                p2["B->D^*::alpha^T23_1@BSZ2015"] = 0.0;
+                p2["B->D^*::alpha^T23_2@BSZ2015"] = 0.0;
+                p2["mass::B_d"]                   = 5.279;
+                p2["mass::D^*_d"]                 = 2.0103;
+                // fix scale
+                p2["mu"]                          = 4.18;
+                // mb(mb)
+                p2["mass::b(MSbar)"]              = 4.18;
+                // mc(mc)
+                p2["mass::c"]                     =  1.275;
+                p2["b->cmunumu::Re{cVL}"]         =  1.0;
+                p2["b->cmunumu::Im{cVL}"]         = -1.0;
+                p2["b->cmunumu::Re{cVR}"]         =  2.0;
+                p2["b->cmunumu::Im{cVR}"]         = -2.0;
+                p2["b->cmunumu::Re{cSL}"]         =  3.0;
+                p2["b->cmunumu::Im{cSL}"]         = -3.0;
+                p2["b->cmunumu::Re{cSR}"]         =  4.0;
+                p2["b->cmunumu::Im{cSR}"]         = -4.0;
```

Comment:
What about C_T?

Reply:
C_T is not here because the NP contribution stemming from C_T is simply neglected in the paper 1307.6653.